app/feature_defaults_unittest.cc

```diff
@@ -214,7 +213,6 @@ TEST(FeatureDefaultsTest, DisabledFeatures) {
       &optimization_guide::features::kOptimizationGuideFetchingForSRP,
       &optimization_guide::features::kOptimizationGuidePersonalizedFetching,
       &optimization_guide::features::kOptimizationHints,
-      &page_image_service::kImageService,
```

Comment:
when feature flags are removed it would be helpful to combine this commit with whatever changes we made to continue to disable the relevant feature or an explanation of why we don't have to